server/middleware/auth.js

```diff
@@ -0,0 +1,25 @@
+const jwt = require('jsonwebtoken');
+
+const User = require('../models/user');
+
+const auth = async (req, res, next) => {
+	try {
+		const token = req.cookies.accessToken;
+		const deciphered = jwt.verify(token, process.env.JWT_SECRET);
+		const user = await User.findOne({
+			_id: deciphered._id,
+		});
+
+		if (!user) {
+			throw new Error();
+		}
+
+		req.user = user;
+		res.cookie('accessToken', token, { httpOnly: true }); //!!!** Is maxAge required? **!!!//
```

Comment:
I think it's a good idea to have an expiration. Maybe 1 month?

Reply:
Yes I agree. I will set it so.